sdk/storage/azure-storage-queue/azure/storage/queue/aio/_queue_client_async.py

```diff
@@ -131,20 +218,18 @@ async def create_queue(self, **kwargs):
                 :dedent: 12
                 :caption: Create a queue.
         """
-        metadata = kwargs.pop('metadata', None)
         timeout = kwargs.pop('timeout', None)
         headers = kwargs.pop("headers", {})
-        headers.update(add_metadata_headers(metadata))  # type: ignore
+        headers.update(add_metadata_headers(metadata))
         try:
-            return await self._client.queue.create(  # type: ignore
+            return await self._client.queue.create(
                 metadata=metadata, timeout=timeout, headers=headers, cls=deserialize_queue_creation, **kwargs
             )
         except HttpResponseError as error:
             process_storage_error(error)
 
     @distributed_trace_async
-    async def delete_queue(self, **kwargs):
-        # type: (Optional[Any]) -> None
+    async def delete_queue(self, **kwargs: Any) -> None:  # type: ignore[override]
```

Comment:
[We](url) can remove `type: ignore[override]` in this file now right? Maybe not since I feel like I made this comment before :)